consoleme/lib/aws.py

```diff
@@ -1770,3 +1770,64 @@ async def normalize_policies(policies: List[Any]) -> List[Any]:
                     modified_elements.add(policy[element][i])
             policy[element] = sorted(modified_elements)
     return policies
+
+
+def allowed_to_sync_role(
+    role_arn: str, role_tags: List[Optional[Dict[str, str]]]
+) -> bool:
+    """
+    This function determines whether ConsoleMe is allowed to sync or otherwise manipulate an IAM role. By default,
+    ConsoleMe will sync all roles that it can get its grubby little hands on. However, ConsoleMe administrators can tell
+    ConsoleMe to only sync roles with either 1) Specific ARNs, or 2) Specific tag key/value pairs. Only one allowed_tag
+    has to be configured in the role for ConsoleMe to be allowed to sync.
```

Comment:
> Only one allowed_tag
    has to be configured in the role for ConsoleMe to be allowed to sync.

For this line, if there are 2 tag key/value pairs in the config, and the role only contains one of them, it won't allow the role. Meaning that _all_ allowed_tag key/value pairs need to be present in the role in order for the role to pass. So I think we may want to make this explicit?